web-console/knikubevirt/migrate-vm/migrate-vm.md

```diff
@@ -4,13 +4,17 @@ Clicking the action button next to any Virtual Machine in the List will present
 
 ![Selecting a VM](img/1-0-list.png)
 
-A modal will be presented where the user can choose which Node the VM should be migrated to.
+Since the user can't pick a destination node (we just submit a request for migration and this starts a machinery which at some points picks a host), so we just keep the confirmation dialog simple and let the user know that this VM is migrated to another node.
 
 ![Modal migrate](img/1-2-modal-migrate.png)
```

Comment:
I wonder if we should just say `vm-name-1 will be migrated to the next most suitable node.` Then we don't need a tooltip. @jelkosz is that the proper statement?